security/acme-client/src/opnsense/mvc/app/models/OPNsense/AcmeClient/AcmeClient.xml

```diff
@@ -935,6 +936,12 @@
                         <sha1>SHA1</sha1>
                     </OptionValues>
                 </dns_kas_authtype>
+                <dns_dedyn_token type="TextField">
+                    <Required>N</Required>
+                </dns_dedyn_token>
+                <dns_dedyn_name type="TextField">
+                    <Required>N</Required>
+                </dns_dedyn_name>
```

Comment:
The acme.sh dns hook is "dns_desec", all fields should use this name instead of "dns_dedyn".

Reply:
@fraenki, thanks for review! It is fixed now.